src/Account.ts

```diff
@@ -1,8 +1,12 @@
 import Web3 from 'web3';
```

Comment:
Please check linting issue in this file. Also please check linting issue in test files.

Reply:
this file has no linter errors. We had checked all test files with the previous PR.